gulpfile.ts

```diff
@@ -75,3 +76,17 @@ gulp.task('build_server', () => {
     .js
     .pipe(gulp.dest('out/'));
 });
+
+const mongodb_path = './third_party/mongodb';
+gulp.task('start-mongo', runCommand(`${mongodb_path}/bin/mongod --dbpath ${mongodb_path}`));
```

Comment:
We are already using gulp.shell. So, can we just use gulp.shell() instead of runCommand()?

Reply:
Oh, I didn't know about 'gulp.shell'. I will look into it and fix it.

Reply:
@llighter I had to give you the information early but I couldn't. I'm sorry for that. Also, thank you for this work!

Reply:
@romandev It's fine. I was able to study this in more detail.

Reply:
@romandev I've found that now gulp do not recommend 'gulp.shell' anymore.
Here is [blackList.json file]( https://github.com/gulpjs/plugins/blob/master/src/blackList.json) at gulpjs repo.
Gulp recommend to use [gulp-exec]() or [child_process]().
And according to [gulp-exec]() docs, If you just want to run a command, don't use this plugin.
And [gulp API docs](https://github.com/gulpjs/gulp/blob/master/docs/API.md#gulpdestpath-options) use [child_process](https://github.com/gulpjs/gulp/blob/master/docs/API.md#gulpdestpath-options) like this.
```
// run a command in a shell
var exec = require('child_process').exec;
gulp.task('jekyll', function(cb) {
  // build Jekyll
  exec('jekyll build', function(err) {
    if (err) return cb(err); // return error
    cb(); // finished task
  });
});
```
What should I use?
WDYT? PTAL :)

Reply:
Okay! Thank you for the information. If so, we can move this patch to gulp-exec or child_process.
And, once this patch is merged, you can also replace gulp-shell with gulp-exec or child_process in existing codes.

Reply:
@romandev ok, then I will replace gulp-shell with child_process in our code.